corehq/apps/app_manager/xpath_validator/nodejs/xpathConfig.js

```diff
@@ -0,0 +1,48 @@
+if (typeof require !== 'undefined' && typeof exports !== 'undefined') {
+    var parser = require('js-xpath/parser');
+    var xpath = require('js-xpath/xpath');
+}
+
+var XPATH_CONFIG = (function () {
+    function getAllowedHashtags(allowCaseHashtags) {
+        var replacements = {
+        '#session': "ok",
+        '#user': "ok"
+        };
+        if (allowCaseHashtags) {
+            replacements['#case'] = 'ok';
+            replacements['#parent'] = 'ok';
+            replacements['#host'] = 'ok';
```

Comment:
I believe it's analogous to `#parent` but for extension cases. (Child cases have "parent" cases, extension cases have "host" cases.)